lib/configuration/validation/account.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Portions Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import Joi from 'joi';
+
+export default Joi.object().keys({
+    password: Joi.string().allow(''),
+    current_password: Joi.string().allow('')
+});
```

Comment:
password_confirmation: Joi.any().equal(Joi.ref('password')).required()

Reply:
`password_confirmation` is not expected to be sent to the backend security plugin